app/views/content_items/_context_and_title.html.erb

```diff
@@ -1,26 +1,12 @@
-<%
-  context_string =  t("content_item.schema_name.#{@content_item.document_type}", count: 1);
-  context_inside = false;
- %>
-
-<% @content_item&.featured_attachments.each do |fa| %>
-  <% return if !@content_item.attachment_details(fa).present? %>
-  <% if @content_item.attachment_details(fa)['title'] == @content_item.title %>
-      <% content_for :title do %>
-        <%= t("content_item.schema_name.#{@content_item.document_type}.overview", count: 1) %>: <%= @content_item.title %>
-      <% end %>
-      <%
-         context_string =  t("content_item.schema_name.#{@content_item.document_type}.overview", count: 1) << ":"
-         context_inside = true
-      %>
-      <% break %>
-    <% end %>
+<% content_for :title do %>
+  <%= t("content_item.schema_name.#{@content_item.document_type}", count: 1) %>: <%= @content_item.title %>
+   <% break %>
```

Comment:
The break was initially there to jump out of the each loop once we found a match, since we're no longer in a loop I don't think it's needed.